app/auto_responder_test.go

```diff
@@ -185,14 +213,65 @@ func TestSendAutoResponseSuccess(t *testing.T) {
 	userUpdated1, err := th.App.PatchUser(user.Id, patch, true)
 	require.Nil(t, err)
 
-	th.App.CreatePost(&model.Post{
+	savedPost, _ := th.App.CreatePost(&model.Post{
+		ChannelId: th.BasicChannel.Id,
+		Message:   "zz" + model.NewId() + "a",
+		UserId:    th.BasicUser.Id},
+		th.BasicChannel,
+		false, true)
+
+	sent, err := th.App.SendAutoResponse(th.BasicChannel, userUpdated1, savedPost)
+
+	assert.Nil(t, err)
+	assert.True(t, sent)
+
+	if list, err := th.App.GetPosts(th.BasicChannel.Id, 0, 1); err != nil {
+		require.Nil(t, err)
+	} else {
+		autoResponderPostFound := false
+		for _, post := range list.Posts {
+			if post.Type == model.POST_AUTO_RESPONDER {
+				autoResponderPostFound = true
+				assert.Equal(t, savedPost.Id, post.RootId)
+				assert.Equal(t, savedPost.Id, post.ParentId)
+			}
+		}
+		assert.True(t, autoResponderPostFound)
+	}
```

Comment:
Is the `if` needed? Can it be dropped? Can we also change this in the other test?